src/test/java/teammates/test/cases/browsertests/InstructorEditStudentFeedbackPageUiTest.java

```diff
@@ -25,8 +27,12 @@ protected void prepareTestData() {
     @Test
     public void testAll() throws Exception {
         testModerationHint();
+        testAddCommentsToQuestionsWithoutResponse();
         testEditResponse();
         testAddResponse();
+        testAddCommentsToQuestionsWithResponses();
+        testEditCommentsActionAfterAddingComments();
+        testDeleteCommentsActionAfterEditingComments();
```

Comment:
Possible to write separate `@Test`?

Reply:
I guess not, because they are interdependent. It'll become very long and complex.

Reply:
> It'll become very long and complex.

I think no. Use `@BeforeMethod` and write empty body to `prepareTestData` would save you.